### PR TITLE
Small fix

### PR DESCRIPTION
**Pull request by freeman91**
No Story

## Summary

A couple things that threw errors when I was running things

### CODE REVIEW

File: frontend/views/Login.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ function Login(props: Props): React.ReactElement {
   const [password, setPassword] = useState<string>("");
   const [passwordConfirmation, setPasswordConfirmation] = useState<string>("");
   const route = useRoute();
-  const token = route.params ? route.params.token : null;
+  const token = props.route.params ? props.route.params.token || "" : "";
```
Comment (freeman91) [inline frontend/views/Login.tsx:52]: `props.route.params` might be undefined that's why this was throwing an error

File: script/db_workbench.py [hunk 2]
```diff
@@ -6,11 +6,11 @@
 from flask_bcrypt import Bcrypt
 from flask_jwt_extended import JWTManager, create_access_token
 
+sys.path.insert(0, "/appdata")
```
Comment (freeman91) [inline script/db_workbench.py:9]: this is required to import create_app and connect 
Idk if there's a better way to do this though

Comment (NickHackman) [reply script/db_workbench.py:9]: Whoops, my bad this was probably caused by Isort.

File: frontend/views/Login.tsx (hunk 1 above)
Comment (NickHackman) [reply frontend/views/Login.tsx:52]: There has to be a better syntax for this, I'm not really sure why the previous code wouldn't work. If params was undefined then params.token would be undefined and if it was it'd use empty string as a default. 

I guess just make it what it previously was

```ts
const token = props.route.params ? props.route.params.token : "";
```

**Review by NickHackman — APPROVED** [review]
LGTM! Thanks for catching these.

**Pull request merged**